### PR TITLE
Adds support for a Google Cloud SQL sidecar

### LINKED ISSUE #298: Add support for connection to SQL with proxy inside GKE
In GCP the recommended way of connecting to Cloud SQL is by using [Cloud SQL Auth Proxy](https://cloud.google.com/sql/docs/mysql/sql-proxy). This applies to local development as well as inside your GKE cluster.

When hosting LakeFS in GKE, it would be great to have an easy way of connecting to a Cloud SQL instance. 

#297 shows an example of how to do that.

### PR DESCRIPTION
**Pull request by oddeirikigland**
closes #298

### CODE REVIEW

**Comment by itaiad200** [comment]
Hey @oddeirikigland , thank you for your contribution!
Do you mind opening an issue, explaining the motivation behind this change and link the issue to this PR?

**Comment by oddeirikigland** [comment]
> Hey @oddeirikigland , thank you for your contribution! Do you mind opening an issue, explaining the motivation behind this change and link the issue to this PR?

Of course, added a short explanation now @itaiad200

File: charts/lakefs/README.md [hunk 1]
```diff
@@ -130,5 +150,7 @@ helm upgrade -f my-values.yaml my-lakefs lakefs/lakefs --set kv_upgrade=true
...
 | `s3Fallback.aws_secret_key`           | An AWS secret key to be used by the S3Proxy for authentication                                                                                                                                                                                                                       |                                     |
+| `gcpFallback.enabled`                  | If set to true, an [GCP Proxy](https://github.com/GoogleCloudPlatform/cloud-sql-proxy) container will be started.                                                                           |                                     |
+| `gcpFallback.instances`           | The instances to connect to.                                                                                                                                                                                                                       |                                     |
```
Comment (itaiad200) [inline charts/lakefs/README.md:154]: `instances` means there could be more than a single value right? How do you expect them to be passed?

**Review by itaiad200 — CHANGES_REQUESTED** [review]
Thank you for your contribution! Looks great. Few comments:
1. How was this tested?
2. Notice that the CI tests failed, please fix.
3. Notice the single comment on the `instances` attribute.

Please complete the above items, we'd very much like to merge this change 💪

**Review by itaiad200 — APPROVED** [review]
LGTM!

**Pull request merged**